### DISCUSSION TITLE
I am unable to export component using bit remote add

### DISCUSSION THREAD

**Discussion by darshandung** [teambit/bit#6920] [Community]
Hi Team,

I pull bitcli/bit-server from docker hub and i am running through this command run -it -p 5000:3000 bitcli/bit-server:latest.
I can see bit.dev ui on http://localhost:5000. now i am trying to add **bit remote add http://localhost:5000** but it's giving me error

bit remote add http://localhost:5000/
{
  "response": {
    "error": "<!DOCTYPE html>\n<html lang=\"en\">\n<head>\n<meta charset=\"utf-8\">\n<title>Error</title>\n</head>\n<body>\n<pre>Cannot POST //graphql</pre>\n</body>\n</html>\n",
    "status": 404
  },
  "request": {
    "query": "\n      {\n        scope {\n          name\n        }\n      }\n    "
  }
}

Can someone please help me , i want to export component on http://localhost:5000/

**Comment by beaupd** [comment]
The problem is the trailing slash in the url. "http://localhost:5000/" should be "http://localhost:5000"
